Automattic/woocommerce-payments
# refactor: remove sortable payment methods list

Fixes #

#### Changes proposed in this Pull Request









UI has changed
- Removed the `@dnd-kit/*` sortable library dependencies
- Payment methods list does not have drag handles anymore
- Payment methods list does not have "Manage" link anymore
- Payment methods list has been renamed from `OrderableList` to `PaymentMethodsList`
- Removed unused styles from the list

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout

Before:
![2021-05-12 15 29 46](https://user-images.githubusercontent.com/273592/118040229-ec5c2600-b336-11eb-81e7-f1457461f713.gif)

After:
![2021-05-12 16 33 01](https://user-images.githubusercontent.com/273592/118047002-bf604100-b33f-11eb-9db5-17bccfc4bbb2.gif)


-------------------
